File: Inciante/Idade em Dias.py
idade_dias=int(input())
anos=idade_dias//365
if anos==0:
    meses=idade_dias//30
elif anos !=0:
    meses=(idade_dias - anos*365)//30

if meses==0:
    dias=idade_dias
elif meses!=0:
    dias=(idade_dias - anos*365 - meses*30)

print(anos, "ano(s)")
print(meses, "mes(es)")
print(dias, "dia(s)")

File: Iniciante C/Idade em Dias.c
#include <stdio.h>
int main(){
    int idade,meses,dias;
    scanf("%d",&idade);
    int anos = idade/365;
    if (anos==0)
    {
        meses=idade/30;
    }
    if (anos!=0)
    {
        meses=(idade-anos*365)/30;
    }
    if (meses==0)
    {
       dias=idade;
    }
    if (meses!=0)
    {
        dias=(idade-anos*365-meses*30);
    }
    printf("%d ano(s)\n", anos);
    printf("%d mes(es)\n", meses);
    printf("%d dia(s)\n", dias);
    return 0;
}